appveyor.yml

```diff
@@ -1,23 +1,15 @@
 version: 4.0.0.{build}
 
-os: Visual Studio 2019
-
 branches:
   except:
     - configdata
     - gh-pages
 
-environment:
-  # The following entries define the default values of the environment variables.
-  # They can be overridden in AppVeyor | Settings | Environment.
```

Comment:
This comment is important IMO if we keep this section.

Reply:
This whole section is gone...

Reply:
We can't have it without something defined
![image](https://user-images.githubusercontent.com/4403806/76164044-a12d0d00-619f-11ea-8830-18d5ae131ec6.png)